### PR TITLE
[AL-2659] Inclusion of optional metadata parameter

### PR DESCRIPTION
**Pull request by jtsodapop**

### CODE REVIEW

File: labelbox/schema/batch.py [hunk 1]
```diff
@@ -92,23 +94,27 @@ def export_data_rows(self, timeout_seconds=120) -> Generator:
...
                 response.raise_for_status()
                 reader = ndjson.reader(StringIO(response.text))
-                # TODO: Update result to parse metadataFields when resolver returns
-                return (Entity.DataRow(self.client, {
-                    **result, 'metadataFields': [],
-                    'customMetadata': []
-                }) for result in reader)
+                return (Entity.DataRow(
+                    self.client, {
+                        **result, 'customMetadata': result['metadata'],
```
Comment (kkim-labelbox) [inline labelbox/schema/batch.py:115]: I think this should be `result['customMetadata']`, but can you double check by printing the result?

**Review by kkim-labelbox — COMMENTED** [review]
LGTM! One comment

File: labelbox/schema/batch.py (hunk 1 above)
Comment (jtsodapop) [reply labelbox/schema/batch.py:115]: `DataRow {'created_at': datetime.datetime(2022, 6, 1, 21, 13, 28, tzinfo=datetime.timezone.utc), 'external_id': '5668a62497e6edcd', 'media_attributes': {}, 'metadata': [DataRowMetadataField(schema_id='cl3w2pwsr000nj6ouhqebbf19', value=datetime.datetime(2021, 10, 13, 21, 1, 44, 34911, tzinfo=datetime.timezone.utc))], 'metadata_fields': [{'schemaId': 'cl3w2pwsr000nj6ouhqebbf19', 'name': 'capture_date', 'kind': 'CustomMetadataDateTime', 'value': '2021-10-13T21:01:44.034911Z'}], 'row_data': 'https://storage.googleapis.com/labelbox-datasets/open-images/data/5668a62497e6edcd.jpg', 'uid': 'cl3w355rxh5doj6ou1ftpffq7', 'updated_at': datetime.datetime(2022, 6, 1, 21, 13, 28, tzinfo=datetime.timezone.utc)}> `

This is an example result. i think we said that customMetadata -> `metadata` which should be the flattened view while `metadata_fields` is the simplified version

**Review by kkim-labelbox — APPROVED** [review]
LGTM!

**Pull request merged**